source/_data/projects.yml

```diff
@@ -0,0 +1,33 @@
+# open source projects that use Cypress to write end-to-end tests
+
+- name: Redash
+  description: "Connect and query your data sources, build dashboards to visualize data and share them with your company."
+  url: https://github.com/getredash/redash/tree/master/client
+
+- name: Mermaid.js
+  description: "Generation of diagram and flowchart from text in a similar manner as markdown"
+  url: https://github.com/mermaid-js/mermaid
+
+- name: WithSpectrum
+  description: "Simple, powerful online communities at spectrum.chat"
+  url: https://github.com/withspectrum/spectrum
+
+- name: Vue Storefront
+  description: "PWA for eCommerce that is 100% offline, platform agnostic, and headless"
+  url: https://github.com/DivanteLtd/vue-storefront/tree/master/test/e2e
+
+- name: Netlify CMS
+  description: "A CMS for Static Site Generators at netlifycms.org"
+  url: https://github.com/netlify/netlify-cms/tree/master/cypress
+
+- name: ngx-bootstrap
+  description: Fast and reliable Bootstrap widgets in Angular
+  url: https://github.com/valor-software/ngx-bootstrap/tree/development/cypress
+
+- name: "octokit/rest.js"
```

Comment:
```suggestion
- name: "rest.js"
```

Reply:
I would keep Oktokit - it makese it clear that this is GitHub's REST client library